src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -4,10 +4,10 @@
 import logging
 import time
 
-from ceph.deployment.drive_group import DriveGroupSpec, DriveGroupValidationError
+from ceph.deployment.drive_group import DriveGroupSpec, DriveGroupValidationError  # type: ignore
```

Comment:
Wasn't sure what to do about the mypy error here, #type: ignore fixes it for now